.github/workflows/check.yaml

```
@@ -48,3 +48,14 @@ jobs:
           # cause those comments get rendered in documentation by godoc.
           # See https://github.com/tarantool/go-tarantool/pull/160#discussion_r858608221
           args: -E goimports -D errcheck
+
+  codespell:
+    runs-on: ubuntu-latest
```

Oh, I've noticed a little problem: the job runs two times for a pull request. Could you make this change? See `luacheck` and `golangci-lint` jobs in the file.
```suggestion
    runs-on: ubuntu-latest
    if: |
      github.event_name == 'push' ||
      github.event_name == 'pull_request' &&
      github.event.pull_request.head.repo.full_name != github.repository
```

---

Thank you! Fixed.